addons/shaderV/examples/basic_examples.tscn

```diff
@@ -1592,7 +1592,7 @@ input_name = "uv"
 input_name = "time"
 
 [sub_resource type="VisualShaderNodeCustom" id="VisualShaderNodeCustom_swrn3"]
-default_input_values = [0, Vector3(0, 0, 0), 1, 1.0, 2, Vector3(0, 0, 0)]
```

Comment:
There were invalid default values from previous iteration stored in the scene. It should be fine now.